otter/models/cass.py

```diff
@@ -808,7 +809,24 @@ def _do_update(ts, _):
                  'status': status.name},
                 DEFAULT_CONSISTENCY)
 
-        return self.view_config().addCallback(_do_update)
+        @self.with_timestamp
+        def set_deleting(ts, _):
```

Comment:
Given that we're not using the config information here (`_`), we should be able to avoid calling `view_config` in this code path entirely, I think?